README.md

```diff
@@ -205,6 +242,13 @@ Example:
 
 > Mention all of the collaborators of this project.
 
+<<<<<<< HEAD
+self project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+=======
 👤 **Author1**
 
 - GitHub: [@githubhandle](https://github.com/githubhandle)
```

Comment:
Your social handles are not edited with your details. Kindly edit them to personalise your work 👏